SocialCareCaseViewerApi.Tests/V1/UseCase/FormSubmissionUseCaseTests.cs

```diff
@@ -389,6 +408,30 @@ public void UpdateAnswersSuccessfullyChangesSubmissionAnswers()
             _mockMongoGateway.Verify(x => x.UpsertRecord(CollectionName, createdSubmission.SubmissionId, It.IsAny<CaseSubmission>()), Times.Once);
         }
 
+        private static object[] _invalidSubmissionStateForUpdates =
+        {
+            SubmissionState.Discarded,
+            SubmissionState.Submitted,
+            SubmissionState.Approved
+        };
```

Comment:
Could this be moved to the top of the file beside other variables? Since it's used by another test further up.

Reply:
Yes, good point :+1: